src/styles/blocks/exhibition.scss

```diff
@@ -0,0 +1,140 @@
+.exhibition {
+
+  &__photos {
+    display: flex;
+    margin-top: 48px;
+    gap: 20px;
+
+    @include onTablet {
+      gap: 24px;
+    }
+  }
+
+  &__photo {
+    height: 130px;
+    background-size: contain;
+    background-position: center;
+    background-repeat: no-repeat;
+
+    @include onTablet {
+      height: 186px;
+    }
+
+    @include onDesktop {
+      height: 320px;
+    }
+  }
+
+  &__photo--1 {
+    background-image: url(../images/photo-1-small.png);
```

Comment:
Its better to have photos in <img> tag because of accessibility issues.
Useful link: [HERE](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image)

Reply:
In this section I have different images on same places depend on screen width, so I don't use here tag images